src/components/soknadpanel/SporsmalVarianter/Tall.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { BodyShort, Heading } from '@navikt/ds-react';
+
+import { cleanId } from '../../../utils/stringUtils';
+import { SoknadSporsmalFragment, SoknadSporsmalSvartypeEnum } from '../../../graphql/queries/graphql.generated';
+
+import { SporsmalVarianterProps } from './SporsmalVarianter';
+// eslint-disable-next-line postcss-modules/no-unused-class
+import styles from './SporsmalVarianter.module.css';
+
+function Tall({ sporsmal }: SporsmalVarianterProps): JSX.Element | null {
+    const listItemId = cleanId(sporsmal.sporsmalstekst);
+    const nestedListItemId = cleanId('nested-list-item-id');
+
+    function setLabel(sporsmal: SoknadSporsmalFragment): string {
```

Comment:
denne ser ut som en pure function, den kan du putte utenfor komponenten, kanskje kalle den for getLabel?